code/modules/atmospherics/machinery/pipe.dm

```diff
@@ -614,6 +622,7 @@ obj/machinery/atmospherics/pipe
 		level = 2
 		alpha = 255
 		fatigue_pressure = INFINITY
+		big_pipe = JUNCTION_SIZE // bypasses some checks
```

Comment:
why are you setting a TRUE/FALSE to a number?

Reply:
originally i had it as true/false but then made it so if JUNCTION_SIZE was passed in, the pipes would ignore the size difference

Reply:
it should be 0 or 1 now, i just changed it